### ISSUE TITLE
Colorpicker defaults in fieldset not working

### ISSUE THREAD

**Issue by Tekanewa** [Codestar/codestar-framework#374]
As the title states, when using colorpickers within a fieldset, the default values for the colorpickers don't get applied. The value for each colorpicker is simply empty.

**Comment by Codevz** [comment]
Its working fine for me with this code: 

```PHP
    array(
      'id'      => 'color_picker_1',
      'type'    => 'color_picker',
      'title'   => 'Color Picker',
      'default' => '#dd3333',
    ),
```

**Comment by Tekanewa** [comment]
Yes, with a single color picker, I don't have any issues; the problem is when using (multiple) colorpickers within a fieldset:

`array(
    'id'        => 'fieldset_1',
    'type'      => 'fieldset',
    'title'     => 'Fieldset Field',
    'fields'    => array(  
        array(
            'id'      => 'color_picker_1',
            'type'    => 'color_picker',
            'title'   => 'Color Picker',
            'default' => '#ffffff',
        ),
        array(
            'id'      => 'color_picker_2',
            'type'    => 'color_picker',
            'title'   => 'Color Picker 2',
            'default' => '#37404e',
        ),
        array(
            'id'      => 'color_picker_3',
            'type'    => 'color_picker',
            'title'   => 'Color Picker 3',
            'default' => '#777678',
        ),
    ),
),`

None of the default values are set on my end (no errors in the browser console either). The "data-default-color" attribute is correctly set on the input, but the "value" attribute is empty, and so is the input box for each colorpicker.

**Comment by Codevz** [comment]
You are right. I think this should fix the problem, Edit cs-framework > fields > color_picker > color_picker.php and change output function to this: 

```PHP
  public function output() {

    $value = $this->element_value() ? $this->element_value() : $this->field['default'];

    echo $this->element_before();
    echo '<input type="text" name="'. $this->element_name() .'" value="'. $value .'"'. $this->element_class( 'cs-field-color-picker' ) . $this->element_attributes( $this->extra_attributes() ) .'/>';
    echo $this->element_after();

  }
```

**Comment by Tekanewa** [comment]
This will correctly set the value attribute on the input, but after the colorpicker routine is applied to the input, the input field still remains empty on my end, and the colorpicker is also not picking it up.

**Comment by Codestar** [comment]
Interesting, this code working normally in my local ? maybe there is `id` conflict.  `'id' => 'unique_1',` try to change ids for test.

**Comment by Tekanewa** [comment]
For test purposes, I'm using the exact code as I showed in my prior comment, where each colorpicker has an unique ID (color_picker_1/color_picker_2/color_picker_3). I also tested it with the ID's I'm actually wanting to use for my post type, with the same results.
